roles/kubernetes-apps/ansible/templates/kubedns-rc.yml

```diff
@@ -8,7 +8,7 @@ metadata:
     version: v19
     kubernetes.io/cluster-service: "true"
 spec:
-  replicas: 1
+  replicas: {{ dns_replicas }}
```

Comment:
It is here https://github.com/kubernetes-incubator/kargo/pull/652/files#diff-f5373391df55626f1cdb1241d52fad69R11 and IIUC, those are atomic for an ansible playbook (can't be separated). Not sure if duplicating defined defaults makes sens. This only brings confusion with different defaults might be slipping into the code, if changed in one place only.